TagsCloudContainer/Common/SimpleTag.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections.Generic;
+using TagsCloudVisualization;
+
+namespace TagCloudContainerTests
+{
+    public class SimpleTag
+    {
+        public readonly string Word;
+        public readonly int Count;
+        public Palette  Palette { get; set; }
```

Comment:
```suggestion
        public Palette Palette { get; set; }
```
Не забывай форматировать код перед отправкой)